src/main/assets/js/hub-page-disable-upon-submit.ts

```diff
@@ -0,0 +1,13 @@
+import { HUB_PAGE_DISABLE_UPON_SUBMIT } from '../../steps/common/content.utils';
+
+import { getById } from './selectors';
+
+const hubPageDisableUponSubmit = (): void => {
+  const submitButton = document.getElementsByClassName(HUB_PAGE_DISABLE_UPON_SUBMIT)[0] as Element | null;
+  const form = getById('main-form') as HTMLFormElement | null;
+  if (form && submitButton) {
+    form.addEventListener('submit', () => submitButton.classList.add('hub-draft-conditional-order-submitted'));
+  }
+};
+
+hubPageDisableUponSubmit();
```

Comment:
not sure about this but maybe might be worth calling these functions on page load or another event that waits for the elements to appear before trying to get the elements on the page.